nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-ldap/src/main/java/org/nuxeo/ecm/directory/ldap/LDAPSession.java

```diff
@@ -89,6 +89,9 @@
 
     private static final Log log = LogFactory.getLog(LDAPSession.class);
 
+    // set to false for debugging
+    private static final boolean HIDE_PASSWORD_IN_LOGS = true;
```

Comment:
When debugging low level stuff it's sometimes useful to confirm what password has been read from the database in traces. Note that you have to modify the source to change it so it's no big deal I hope.